packages/appstore/src/components/add-more-wallets/carousel-buttons.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+
+type PrevNextButtonProps = {
+    enabled: boolean;
+    onClick: () => void;
+};
+
+export const PrevButton: React.FC<PrevNextButtonProps> = props => {
```

Comment:
I guess we don't use `React.FC` in our application, correct me if I'm wrong

Reply:
yup no need